integration/app/test-demo/src/pages/VideoFilterTestApp.tsx

```diff
@@ -0,0 +1,115 @@
+
+  const setBlurStrength = (newBlurStrength: number): void => {
+    console.log(`Setting blur strength to ${newBlurStrength} - should re-initialize the bg blur provider`);
+    const options = {...backgroundBlurOptions, blurStrength : newBlurStrength};
+    setBackgroundBlurOptions(options);
+  }
+
+  return(
+    <div style={{ display: 'flex', flexDirection: 'column', width: '70%', margin: 'auto' }}>
+      <BackgroundBlurProvider options={backgroundBlurOptions}>
+        <MeetingProvider {...{logLevel: LogLevel.INFO}}>
+          <h3 data-testid='app-name'>Video Filter Test</h3>
+          <Meeting onBlurStrengthChanged={(newBlurStrength) => setBlurStrength(newBlurStrength)}/>
+        </MeetingProvider>
+      </BackgroundBlurProvider>
+    </div>
+    )
+};
+
+interface Props {
+  onBlurStrengthChanged: (blurStrength: number) => void;
+}
+
+const Meeting: React.FC<Props> = ({ onBlurStrengthChanged }) => {
+  const { isBackgroundBlurSupported, createBackgroundBlurDevice } = useBackgroundBlur();
+  const meetingManager = useMeetingManager();
+  const { selectedDevice } = useVideoInputs();
+  const { toggleVideo } = useLocalVideo();
+  const meetingStatus = useMeetingStatus();
+  const [ blurStrength, setBlurStrength ] = useState<string>("20");
+
+  useEffect(() => {
+    const addBackgroundBlur = async () => {
+      if (
+        isBackgroundBlurSupported &&
+        meetingStatus === MeetingStatus.Succeeded
+      ) {
+        console.log("Creating background blur device called");
+        const chosenVideoTransformDevice = await createBackgroundBlurDevice(selectedDevice);
+        console.log(chosenVideoTransformDevice);
+        await meetingManager.selectVideoInputDevice(chosenVideoTransformDevice);
+        toggleVideo();
+      }
+    };
+    addBackgroundBlur();
+  }, [
+    isBackgroundBlurSupported,
+    meetingStatus,
+  ]);
+
+  const updateBlurStrength = (e: ChangeEvent<HTMLSelectElement>) => {
+    e.preventDefault();
+    setBlurStrength(e.target.value);
+    console.log('Updated to blurStrength:' + e.target.value);
+    const newBlurStrength = parseInt(e.target.value);
+    onBlurStrengthChanged(newBlurStrength);
+  }
+
+  return (
+    meetingStatus !== MeetingStatus.Succeeded ?
+    <MeetingForm /> :
+    <>
+      <div style={{ display: 'flex', flexDirection: 'column', width: '70%', margin: 'auto' }}>
+      <MeetingInfo />
+      <div style={{ height: '30rem', width:'100%' }}>
+        <VideoTileGrid />
+      </div>
+      <ControlBar
+        layout='undocked-horizontal'
+        css='margin-top: 1rem;'
+        showLabels
+      >
+        <VideoInputBackgroundBlurControl />
+        <FormField
+          field={Select}
+          label="Blur Strength"
+          value={blurStrength}
+          onChange={updateBlurStrength}
+          options={[{value: '20', label: '20'}, {value: '40', label: '40'}, {value: '60', label: '60'}]}
+        />
+        <MeetingLeaveControl />
+      </ControlBar>
+      </div>
+    </>
```

Comment:
Shall we change the order of components to below to keep the style consistent with roster test app?
```
<MeetingForm />
<MeetingInfo />
<ControlBar />
<VideoTileGrid />
```

Reply:
I've left the MeetingInfo at the top, I dont think it needs to be the exact same, does it? Different tests will have different structures.